packages/ui5-library-reference-inquirer/README.md

```diff
@@ -49,7 +49,9 @@ export default class UI5LibraryReferenceGenerator extends Generator {
 
         try {
             await generate(this.answers.targetFolder, reuseLibConfigs);
-        } catch (e) {}
+        } catch (e) {
+            console.log(`Error adding reference ${e}`)
```

Comment:
Should be using either the generator logger or preferably the open ux tools logger.

Reply:
I had made the same comment but reverted it as it was a README! And I guess its whoever is implementing the logic would be responsible for the logger?